src/interactive-window/editor-integration/cellFactory.ts

```diff
@@ -1,18 +1,68 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 'use strict';
-import '../common/extensions';
+import '../../platform/common/extensions';
 
+import type * as nbformat from '@jupyterlab/nbformat';
 import { NotebookCell, NotebookCellKind, NotebookDocument, Range, TextDocument, Uri } from 'vscode';
 
-import { appendLineFeed, parseForComments } from '../../datascience-ui/common';
-import { createCodeCell, createMarkdownCell, uncommentMagicCommands } from '../../datascience-ui/common/cellFactory';
-import { IJupyterSettings, Resource } from '../common/types';
-import { noop } from '../common/utils/misc';
+import { appendLineFeed, generateMarkdownFromCodeLines, parseForComments } from '../../webviews/webview-side/common';
 import { CellMatcher } from './cellMatcher';
-import { ICell, ICellRange } from './types';
 import { getInteractiveCellMetadata } from '../../interactive-window/interactiveWindow';
 import { createJupyterCellFromVSCNotebookCell } from '../../notebooks/helpers';
+import { ICell, ICellRange, IJupyterSettings, Resource } from '../../platform/common/types';
+import { noop } from '../../platform/common/utils/misc';
+
+export function createCodeCell(): nbformat.ICodeCell;
```

Comment:
THis is some of the code that was in the old `datascience-ui/common/cellFactory.ts`